<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connect_Dialog</class>
 <widget class="QDialog" name="Connect_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string>坦克客户端连接选择器</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>服务器IP  :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_ip">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>服务器TcpPort:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_port">
       <property name="text">
        <string>8888</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>服务器UdpPort:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_server_udp_port">
       <property name="text">
        <string>6666</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>客户端Udp监听Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="client_udp_listen_port">
       <property name="text">
        <string>6548</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_ok">
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
